### PR TITLE
README enhancements

### PR DESCRIPTION
**Pull request by xsuchy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,7 +22,7 @@ To analyze a log file, run the script with the following command line arguments:
 
 Example usage:
 
-    logdetective https://example.com/logs.txt --model https://mymodel.co/model_file --summarizer drain --n_lines 10
+    ~/.local/bin/logdetective https://example.com/logs.txt
```
[the remark this answers is not in the document]
Comment (xsuchy) [reply README.md:25]: I do not believe that adding  ~/.local/bin/ to PATH is common practice. As any my process can write that I do not completely trust this directory :)

[the remark this answers is not in the document]
Comment (jpodivin) [reply README.md:25]: It's default when you are installing python packages in user space.[0] That way you can avoid giving them root access or messing up one of the many things depending on python in modern Linux.

Alternatively you would have to install it with `sudo pip install .` which is not a good idea. 

[0]https://pip.pypa.io/en/stable/cli/pip_install/#cmdoption-user

[the remark this answers is not in the document]
Comment (xsuchy) [reply README.md:25]: Yes. But this command does not modify my `$PATH`.

[the remark this answers is not in the document]
Comment (jpodivin) [reply README.md:25]: I guess that makes sense. But it seems odd because my path is set to that by default.

**Review by jpodivin — APPROVED** [review]

**Pull request merged**